com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/AdapterUtils.java

```diff
@@ -221,6 +223,14 @@ public static Response setErrorResponse(Response response, ErrorCode errorCode,
         return CompletableFuture.completedFuture(setErrorResponse(response, errorCode, e));
     }
 
+    public static CompletionException createCompletionException(String message, Throwable cause, ErrorCode errorCode) {
```

Comment:
For function overload, you should keep the arguments consistent for the existing order. 

Here, the first two should be message, errorCode in each overload method.

Reply:
Nice catch. thanks.